src/handler.ts

```diff
@@ -0,0 +1,195 @@
+import * as assert from "assert";
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { DynamoDB } from "aws-sdk";
+import * as uuid from "uuid/v4";
+import getConfig from "./getConfig";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+
+type Container = {
+  config: {
+    DynamoDB: object;
```

Comment:
Can you touch on why you typed this as a generic object rather than a specific interface?

Reply:
Glad you called this out.  In the last typescript project I worked on, it was for a client who's tech lead had the final say on what code got merged.  This person insisted that we use `type` instead of `interface`, so this definition is the result of my haste.  I will change to interface as they are more flexible.